src/styles/blocks.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  margin: 0;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gapSize;
+  width: $calSize;
+  padding: 10px;
+
+  &__cells {
+    display: flex;
+    justify-content: center;
+    align-items: center;
```

Comment:
You can also use line-height and text-align to center the text inside cells

Reply:
changes added